licenses.yaml

```diff
@@ -136,6 +136,16 @@ source_paths:
 
 ---
 
+name: code adapted from Apache Knox KnoxSessionStore and ConfigurableEncryptor
+license_category: source
+module: java-core
```

Comment:
Hmm, probably this should be `extensions/druid-pac4j`. I'm not sure why we started to specify the module name in the license file in the first place even though libraries can be used in multiple modules. Per my understanding, the idea is we prioritize the `java-core` module if the library is used in the core as well as extensions. If the library is used in only an extension, it should be the extension name. See https://github.com/apache/druid/blob/master/licenses.yaml#L2954 as an example.

Reply:
Maybe we should talk about why we need the module name here and remove it if possible. But, would you fix the module names properly in this PR?

Reply:
I am not too familiar with `licenses.yaml` file , I only did the changes based on copy-paste and to make build pass.
It  would be great to document somewhere more details about this file, what it does and how it is used  for verification etc.

I agree that when a dep is added, it could  be used  anywhere, so no need  to add a specific   module attribute  if  possible.

Reply:
Yeah, I will create a PR to remove them soon. And maybe write some docs for how to add a new item to `licenses.yaml`.